### ISSUE TITLE
Release 12.0 defaults `enable_system_settings` to true

### ISSUE THREAD

**Issue by aquarapid** [vitessio/vitess#9125]
While we have been defaulting the vtgate reserved connections feature setting
of `enable_system_settings` to true in development versions for a while, we have
been disabling it in release versions.  The 12.0 release binaries has it defaulting to true:

```
$ vitess-12.0.0-de06fca/bin/vtgate -version
Version: 12.0.0 (Git revision de06fca branch 'HEAD') built on Tue Oct 26 11:54:28 UTC 2021 by runner@fv-az198-353 using go1.17.2 linux/amd64
$ vitess-12.0.0-de06fca/bin/vtgate -h 2>&1 | grep system_se -A 1
  -enable_system_settings
        This will enable the system settings to be changed per session at the database connection level (default true)
```

For comparison, 11.0 (11.0.1):

```
$ vitess-11.0.1-92ac1ff/bin/vtgate -h 2>&1 | grep system_se -A 1
  -enable_system_settings
        This will enable the system settings to be changed per session at the database connection level
```

I am not sure this was intentional or an oversight. At a minimum, it should
be featured prominently in the release notes.

**Comment by deepthi** [comment]
This was intentional. We do need to edit the release notes to highlight the change.

**Comment by deepthi** [comment]
Fixed in #9144 #9168.
The release has been edited with the appropriate notes.
